components/app/Modal.vue

```diff
@@ -10,6 +12,16 @@ const closeModal = () => {
   modal.value = false
 }
 
+watch(modal, (value) => {
```

Comment:
it would better to use `:class` with tailwind classes on the modal

Reply:
These styles are targeted to the `body` element, can you clarify more what you mean?

Reply:
Ah yes sorry I thought we are using a different approach, this works! thanks